exonum-java-binding/core/src/main/java/com/exonum/binding/transaction/RawTransaction.java

```diff
@@ -57,6 +57,19 @@
     return new AutoValue_RawTransaction.Builder();
   }
 
+  /**
+   * Creates a raw transaction from the given transaction message.
+   * The returned transaction will have the same service id, transaction id, and payload
+   * as the given message.
+   */
+  public static RawTransaction fromMessage(TransactionMessage txMessage) {
```

Comment:
unit test?

Reply:
It seems too verbose, as it is using a builder, and simple to break, doesn't it?